backend/Models/user.model.js

```diff
@@ -1,38 +1,98 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 
 const Schema = mongoose.Schema;
 
 // Building user schema and defining the fields
-const userSchema = new Schema({
+const userSchema = new Schema(
+  {
     userID: {
-        type: String,
-        unique: true,
-        required: true,
-        trim: true
+      type: String,
+      unique: true,
+      required: true,
+      trim: true,
     },
     refreshToken: {
-        type: String,
-        required: true,
+      type: String,
+      required: true,
     },
     name: {
-        type: String,
+      type: String,
     },
     imageURL: {
-        type: String
+      type: String,
     },
     email: {
-        type: String
+      type: String,
     },
-    groups: [
-        {
-            type: String
+
+    topTracks: [
+      {
+        songName: {
+          type: String,
+          required: true,
+        },
+        songID: {
```

Comment:
I know I left it as songID in the song class but I think trackID might be better because this is what Spotify uses I believe
Not important

Reply:
Sounds good